oak/server/rust/oak_loader/src/main.rs

```diff
@@ -40,6 +44,14 @@ pub struct Opt {
     private_key: Option<String>,
     #[structopt(long, help = "Path to the PEM-encoded certificate chain")]
     cert_chain: Option<String>,
+    #[structopt(
+        long,
+        default_value = "3030",
+        help = "Metrics server port number. Defaults to 3030."
+    )]
+    metrics_port: u16,
+    #[structopt(long, help = "Starts the Runtime without a metrics server.")]
+    no_metrics: bool,
```

Comment:
Sure, but why have a negative meaning for a bool flag? It may as well be positive, and have a default of true. then it can be specified as `--metrics=false` if someone wants to disable the server.

Also see http://go/tott/530 .

Reply:
That makes sense to me if you want to disable the metrics server by default (no `--metrics` option). Then to enable the metrics server one has to use `--metrics` or `--metrics=true` (although the latter looks redundant to me). 
If the server should be disabled by default, then something like `enable-metrics` or `with-metrics` reads better in my opinion.